com.frogatto.Frogatto.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: com.frogatto.Frogatto
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: frogatto
+rename-desktop-file: frogatto.desktop
+rename-appdata-file: frogatto.appdata.xml
+rename-icon: frogatto
```

Comment:
as stated in other PR. if you do provide these files, you should install them with the right name from the ground up. The "rename" stuff is really only for when upstream doesn't conform to XDG standards.